README.md

```diff
@@ -5,7 +5,7 @@
 [![NPM Version](https://img.shields.io/npm/v/monaco-languageclient.svg)](https://www.npmjs.com/package/monaco-languageclient)
 [![NPM Download](https://img.shields.io/npm/dt/monaco-languageclient.svg)](https://www.npmjs.com/package/monaco-languageclient)
 
-NPM module to connect Monaco editor with language servers
+[NPM module](https://www.npmjs.com/) to connect [Monaco editor](https://microsoft.github.io/monaco-editor/) with [language servers](https://langserver.org/)
```

Comment:
The official site for the LSP is https://microsoft.github.io/language-server-protocol/. Please update.

Reply:
Link changed from https://langserver.org/ to https://microsoft.github.io/language-server-protocol/